apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ExperimentsResourceTest.java

```diff
@@ -2688,14 +2690,20 @@ private void fetchAndAssertResponse(boolean userExperimentId, UUID experimentId,
             // then
             assertThat(actualResponse.getStatusInfo().getStatusCode()).isEqualTo(HttpStatus.SC_OK);
             var actualEntity = actualResponse.readEntity(FeedbackScoreNames.class);
-
-            assertThat(actualEntity.scores()).hasSize(expectedNames.size());
-            assertThat(actualEntity
-                    .scores()
-                    .stream()
-                    .map(FeedbackScoreNames.ScoreName::name)
-                    .toList()).containsExactlyInAnyOrderElementsOf(expectedNames);
+            assertFeedbackScoreNames(actualEntity, expectedNames);
         }
+
+        var feedbackScoreNamesByProjectId = projectResourceClient.findFeedbackScoreNames(projectIdsQueryParam, apiKey, workspaceName);
+        assertFeedbackScoreNames(feedbackScoreNamesByProjectId, expectedNames);
```

Comment:
This should be a new test for project resource, to under the experiments resource.

Reply:
I moved it to `ProjectsResourceTest` and did some refactoring.